contracts/vaults/extensions/WrappedPlasmaVault.sol

```diff
@@ -0,0 +1,624 @@
+     * @custom:security Restricted to contract owner
+     */
+    function configurePerformanceFee(address feeAccount_, uint256 feeInPercentage_) external onlyOwner {
+        PlasmaVaultLib.configurePerformanceFee(feeAccount_, feeInPercentage_);
+    }
+
+    /**
+     * @notice Returns the total amount of underlying assets held by the vault
+     * @dev Calculates total assets by querying the maximum withdrawable amount from the underlying PlasmaVault
+     *
+     * Calculation Flow:
+     * 1. Asset Valuation
+     *    - Queries underlying PlasmaVault
+     *    - Determines maximum withdrawable amount
+     *    - Accounts for vault's share position
+     *
+     * 2. Value Representation
+     *    - Returns amount in underlying token decimals
+     *    - Reflects current vault holdings
+     *    - Considers protocol-specific limits
+     *
+     * Integration Context:
+     * - Used for share price calculations
+     * - Affects deposit/withdrawal limits
+     * - Influences fee computations
+     * - Critical for vault operations
+     *
+     * Important Notes:
+     * - Value may fluctuate based on market conditions
+     * - Subject to withdrawal limits
+     * - Considers protocol-specific constraints
+     * - Used in share/asset conversions
+     *
+     * @return uint256 Total assets in underlying token decimals
+     */
+    function totalAssets() public view virtual override returns (uint256) {
+        uint256 grossTotalAssets = ERC4626Upgradeable(PLASMA_VAULT).maxWithdraw(address(this));
+        uint256 unrealizedManagementFee = _getUnrealizedManagementFee(grossTotalAssets);
+
+        if (unrealizedManagementFee >= grossTotalAssets) {
+            return 0;
+        } else {
+            return grossTotalAssets - unrealizedManagementFee;
+        }
+    }
+
+    /**
+     * @notice Returns the current unrealized management fee amount
+     * @dev Calculates accrued management fees since last fee realization using current total assets
+     *
+     * Calculation Flow:
+     * 1. State Assessment
+     *    - Gets current total assets
+     *    - Retrieves fee configuration
+     *    - Checks fee eligibility
+     *
+     * 2. Fee Computation
+     *    - Calculates time elapsed since last update
+     *    - Applies annual fee rate pro-rata
+     *    - Converts to underlying token amount
+     *    - Handles precision and rounding
+     *
+     * Integration Context:
+     * - Used in maxWithdraw calculations
+     * - Affects share price computations
+     * - Impacts withdrawal amounts
+     * - Key for fee tracking
+     *
+     * Important Notes:
+     * - Time-based calculation
+     * - Uses current total assets
+     * - Pro-rata fee computation
+     * - No state modifications
+     *
+     * @return uint256 Unrealized management fee in underlying token decimals
+     */
+    function getUnrealizedManagementFee() public view returns (uint256) {
+        return _getUnrealizedManagementFee(totalAssets());
```

Comment:
don't use totalAssets here because you are doing double-accounting of management fee - use grossTotalAssets